app/Models/Collection.php

```diff
@@ -31,4 +33,46 @@ public function identifiableName()
   {
     return $this->name;
   }
+
+  public static function autoIncrementIdsScript()
+  {
+    $collections = self::withTrashed()->get();
+    $idTracker = 1;
+    $collectionIdMapping = [];
+
+    foreach ($collections as $collection) {
+      $oldId = $collection->id;
+      $collection->id = $idTracker;
+      $collection->save();
+
+      $collectionIdMapping[$oldId] = $idTracker;
+
+      $idTracker++;
+    }
+    return $collectionIdMapping;
+  }
+
+  // $tableName should be snake case, plural string
+  public static function updateCollectionIdInTable($tableName, $collectionIdMapping)
+  {
+    $brokenIds = array();
+    DB::table($tableName)
+      ->whereNotNull('collection_id')
+      ->chunkById(1000, function ($results) use (&$brokenIds, &$collectionIdMapping, $tableName) {
+       foreach ($results as $result) {
+         if (isset($collectionIdMapping[$result->collection_id])) {
+           $newCollectionId = $collectionIdMapping[$result->collection_id];
+         } else {
+           $brokenIds[] = $result->id;
+           continue;
+         }
+
+         DB::table($tableName)
```

Comment:
Could scale better with one update query per collection ID mapping, but this is probably fine in this case.